Loop/Managers/TrustedTimeChecker.swift

```diff
@@ -84,8 +84,8 @@ class TrustedTimeChecker {
     }
 
     private func issueTimeChangedAlert() {
-        let alertTitle = NSLocalizedString("Time Change Detected", comment: "Time change alert title")
-        let alertBody = String(format: NSLocalizedString("Your phone’s time has been changed. %1$@ needs accurate time records to make predictions about your glucose and adjust your insulin accordingly.\n\nCheck in your iPhone Settings (General / Date & Time) and verify that Set Automatically is enabled. Failure to resolve could lead to serious under-delivery or over-delivery of insulin.", comment: "Time change alert body. (1: app name)"), Bundle.main.bundleDisplayName)
+        let alertTitle = NSLocalizedString("iPhone Time Settings Need Attention", comment: "Time change alert title")
```

Comment:
Loop can run on an iPod touch as well. If we do want to refer to the device name here, UIDevice.model can give you that name.